packages-exp/auth-exp/src/core/errors.ts

```diff
@@ -165,7 +165,7 @@ function _debugErrorMap(): ErrorMap<AuthErrorCode> {
     [AuthErrorCode.EMAIL_EXISTS]:
       'The email address is already in use by another account.',
     [AuthErrorCode.EMULATOR_CONFIG_FAILED]:
-      'Auth instance has already been used to make a network call. Auth can ' +
+      '{@link Auth} instance has already been used to make a network call. Auth can ' +
```

Comment:
Here and below: these are not doc strings in this object

Reply:
Oops, reverted.